pkgs/development/python-modules/pyosf/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, pytestrunner
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pyosf";
+  version = "1.0.5";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "psychopy";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fkpmylpcbqa9ky111mz4qr1n8pik49gs7pblbb5qx6b54fzl5k2";
+  };
+
+  preBuild = ''export HOME=$TMP'';
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;  # requires network access
+  checkInputs = [ pytestCheckHook pytestrunner ];
```

if `doCheck` is set to false, `checkInputs` are useless.

In addition, when no unit tests are available you should use `pythonImportsCheck`.

```suggestion
  doCheck = false; # Tests require network access
  pythonImportsCheck = [ "pyosf" ];
```

---

Done!